examples/mvn-hello-world/hello-world/src/main/scala/helloworld/HelloWorldShape.scala

```diff
@@ -0,0 +1,22 @@
+package helloworld
+
+import akka.stream.scaladsl._
+import cloudflow.akkastream._
+import cloudflow.akkastream.scaladsl._
+import cloudflow.streamlets._
+import scala.util.Try
+import scala.concurrent.duration._
+
+class HelloWorldShape extends AkkaStreamlet {
+  val shape = StreamletShape.empty
+
+  def createLogic = new RunnableGraphStreamletLogic() {
+    def runnableGraph = {
+      Source
+        .cycle(() => Iterator.continually("hello world!"))
+        .throttle(1, 1.second)
+        .map(println)
+        .to(Sink.ignore)
+    }
+  }
+}
```

Comment:
May be I am missing something. But aren't we talking about maven support for Java projects ? If I can write Scala, then I can very well use sbt.

Reply:
We want to support Maven builds whatever the target language is going to be, this can easily be in Java ...